main/src/ca/uwaterloo/flix/language/fmt/SimpleType.scala

```diff
@@ -439,6 +439,21 @@ object SimpleType {
             case _ :: _ :: _ => throw new OverAppliedType(t.loc)
           }
 
+        case TypeConstructor.Extensible =>
+          val args = t.typeArguments.map(visit)
```

Comment:
Isn't it better to make this TODO instead of mixing it with Schemas? Now they're displayed the same and can potentially muddle together

Reply:
I made it into a TODO